pages/cookbook/nfts.mdx

```diff
@@ -2,8 +2,205 @@
 
 import { Callout } from 'nextra/components'
 
-<Callout emoji="🚨">
+This page lists common examples of working with [NFTs](https://docs.ton.org/develop/dapps/asset-processing/nfts).
 
-  This page is a stub. [Contributions are welcome!](https://github.com/tact-lang/tact-docs/issues)
+## Accepting nft ownership assign
 
+NFT ownership assigned notification message have the following structure.
+
+```tact
+message(0x05138d91) NFTOwnershipAssigned {
+    previousOwner: Address;
+    forwardPayload: Slice as remaining;
+}
+```
+
+Use [receiver](/book/receive) function to accept notification message.
+
+<Callout type="warning" emoji="⚠️">
+
+  Sender of notification must be validated!
+
+</Callout>
+
+Validation can be done using nft item state init and calculating nft item address.
+NFT Item initial data layout is shown below, but sometimes it may differ.
```

Comment:
```suggestion
Validation can be done using NFT item [`StateInit{:tact}`](/book/expressions#initof) and calculating NFT item address.
Initial data layout of the NFT item is shown below, but sometimes it may differ.
```